src/common/buffer.cc

```diff
@@ -44,8 +44,8 @@ using namespace ceph;
 
 #ifdef BUFFER_DEBUG
 static std::atomic_flag buffer_debug_lock = ATOMIC_FLAG_INIT;
-# define bdout { simple_spin_lock(&buffer_debug_lock); std::cout
-# define bendl std::endl; simple_spin_unlock(&buffer_debug_lock); }
+# define bdout { ceph::spin_lock(&buffer_debug_lock); std::cout
```

Comment:
It *should* be since every bdout has to be paired with a bendl to close the scope, analogous to dout/dendl.